plugins/jobtype/src/azkaban/jobtype/HadoopSecurePigWrapper.java

```diff
@@ -149,6 +155,47 @@ public static void runPigJob(String[] args) throws Exception {
     throw new RuntimeException("Pig job failed.");
   }
 
+  /**
+   * Dump Hadoop counters for each of the M/R jobs in the given PigStats.
+   * 
+   * @param pigStats
+   */
+  private static void dumpHadoopCounters(PigStats pigStats) {
+    try {
+      if (props.getBoolean("pig.dump.hadoopCounter", false)) {
+        if (pigStats != null) {
+          JobGraph jGraph = pigStats.getJobGraph();
+          Iterator<JobStats> iter = jGraph.iterator();
+          while (iter.hasNext()) {
```

Comment:
The counters are at the M/R job level.  I believe only successfully completed M/R jobs will have counters.  Counters for failed M/R jobs are not that useful.